components/gems/Gemfile.lock

```diff
@@ -352,8 +352,8 @@ GEM
     concurrent-ruby (1.1.6)
     cookbook-omnifetch (0.9.1)
       mixlib-archive (>= 0.4, < 2.0)
-    cookstyle (5.23.0)
-      rubocop (= 0.75.1)
+    cookstyle (6.0.19)
```

Comment:
6.3 is out now

Reply:
I pulled in Cookstyle 6.3 and Chefstyle 1.0.2

Reply:
Thanks